### ISSUE TITLE
Error when watching multiple input files

### ISSUE THREAD

**Issue by alexpi** [postcss/postcss-cli#95]
My issue is that when I watch multiple input files, the files are compiled at first, but when I change something in the css, I get a "Error: ENOENT: no such file or directory" error. When I specify a certain input file to watch, it works as expected. I am on 3.0.0-beta.

My css related setup is like that:

package.json:
```
"styles": "mkdirp assets && postcss src/css/*.css -d assets/",
"watch:styles": "npm run styles -- -w",
"build:styles": "npm run styles -- --env production",
```

postcss.config.js:
```
module.exports = (ctx) => ({
	plugins: {
		'stylelint': {
			extends: 'stylelint-config-standard',
			rules: {
				indentation: 'tab',
			}
		},
		'postcss-import': { root: ctx.file.dirname },
		'postcss-cssnext': { browsers: '> 5%' },
		'cssnano': ctx.env === 'production' ? { autoprefixer: false } : false,
		'postcss-browser-reporter': {},
		'postcss-reporter': {}
	}
})
```

**Comment by RyanZim** [comment]
I am unable to reproduce this issue. Please make a reduced-test-case-repo, upload it to GitHub, and post the link here so we can investigate further. Thanks!

**Comment by alexpi** [comment]
Ok, here it is: https://github.com/alexpi/postcss-cli-test
Something I didn't mention, is that I am on Windows 10, and this could raise problems with file paths. For example, the onchange package I am also using, required that file globs must be escaped like that `\"src/css/*.css\"` on Windows. I tried the same thing with postcss-cli, but I am getting the same error.

**Comment by RyanZim** [comment]
Confirmed. I had to quote the globs to reproduce on linux. The issue is our handling of globs in the watcher logic. I have a rough patch, I just need to make sure the change doesn't break anything else.

**Comment by RyanZim** [comment]
PR done: https://github.com/postcss/postcss-cli/pull/97.

**Comment by RyanZim** [comment]
PR merged, patch will be released with v3.0.0, if not before.
